third-party/crowdsec/src/test/java/org/apache/james/CrowdsecExtension.java

```diff
@@ -51,8 +51,8 @@ public void beforeAll(ExtensionContext context) {
     }
 
     @Override
-    public void afterAll(ExtensionContext extensionContext) {
-        crowdsecContainer.stop();
+    public void afterAll(ExtensionContext extensionContext) throws IOException, InterruptedException {
+        crowdsecContainer.execInContainer("cscli", "decision", "delete", "--all");
     }
```

Comment:
```suggestion
    public void afterEach(ExtensionContext extensionContext) throws Exception {
        crowdsecContainer.execInContainer("cscli", "decision", "delete", "--all");
    }
    
    @Override
    public void afterAll(ExtensionContext extensionContext) {
    
    }
```

Otherwise you would not have test isolation between tests of a class.